src/scripts/graph/Header.js

```diff
@@ -121,16 +121,17 @@ export default class Header {
         onAllElementsToggled.dispatch({ collapsed: true });
       });
 
-    const unusedToggle = g.append('text')
+    const unusedToggle = g.append('svg:foreignObject')
       .attr('class', 'timeline__header__hideUnusedProperties')
-      .attr('transform', 'translate(230, -5)')
+      .attr("width", "13px").attr("height", "18px")
+      .attr('transform', 'translate(230, -19)')
       .on('click', () => {
         this.onToggleUnusedProperties.dispatch({ visible: !this.editor.options.showEmptyProperties });
-        updateText.call(this);
+        updateIcon.call(this);
       });
-    updateText.call(this);
-    function updateText() {
-      unusedToggle.text(this.editor.options.showEmptyProperties ? '-' : '+')
+    updateIcon.call(this);
```

Comment:
I like this renaming. Ppl tends to forget to update a name when they change value kinds